content/600-about/200-prisma-docs/20-style-guide/06-image-guidelines.mdx

```diff
@@ -2,26 +2,125 @@
 title: 'Prisma docs image guidelines'
 metaTitle: 'Prisma docs style guide: image guidelines'
 metaDescription: 'This page provides guidelines for images in our docs.'
-tocDepth: 2
+tocDepth: 3
 search: false
 ---
 
 ## Screenshots
 
-- Where possible, use SnagIt to edit screenshots.
-- Always apply a centered minimum shadow to the image. This helps the screenshot stand out on the page.
+Use SnagIt to capture and edit screenshots, and add annotations.
```

Comment:
Non-Snagit screenshots are not acceptable at all? That will make contributing a lot more effort 😿

Reply:
I've reworded the bits about SnagIt so that they sound suggestive and its use being a best practice, but not mandatory.

Would that work?